wiki/Tournaments/MWC/2021_4K/en.md

```diff
@@ -96,6 +96,19 @@ The osu!mania 4K World Cup 2021 is run by the [osu! team](/wiki/People/The_Team)
 | ![][flag_VE] | **Venezuela** | **[\_Yisus\_](https://osu.ppy.sh/users/1489811)**, [Neko-Pan](https://osu.ppy.sh/users/8676532), [xXShyzDy0133Xx](https://osu.ppy.sh/users/13545528), [Edvo](https://osu.ppy.sh/users/8301758), [Doryoku](https://osu.ppy.sh/users/5639725), [Cerelac](https://osu.ppy.sh/users/8065567) |
 | ![][flag_VN] | **Vietnam** | **[MashedPotato](https://osu.ppy.sh/users/10494860)**, [Lottery61](https://osu.ppy.sh/users/13821222), [Asrielly](https://osu.ppy.sh/users/2656302), [CPT\_Sivelia](https://osu.ppy.sh/users/12562107), [TriDoanGaming](https://osu.ppy.sh/users/14009758), [LovelyL](https://osu.ppy.sh/users/14821320) |
 
+## Mappools
+
+### Qualifiers
+
```

Comment:
```suggestion

- FreeMod
```

Reply:
This change was on purpose. I understand that consistency is important but the mappoolers were really adamant in not using that term to describe the pool, since it doesn't represent the pool well.

Reply:
i thought it wasnt on purpose because there are spaces before list items

i will remove them and merge the PR then